test/message-input.test.js

```diff
@@ -58,5 +58,35 @@ describe('message-input', () => {
       keyDownOn(textArea.inputElement, 13, ['shift'], 'Enter');
       expect(spy.called).to.be.false;
     });
+
+    it('should not fire a submit event if value is empty', () => {
+      const spy = sinon.spy();
+      messageInput.addEventListener('submit', spy);
+      button.click();
+      expect(spy.notCalled).to.be.true;
```

Comment:
```suggestion
      expect(spy.called).to.be.false;
```

Let's use this syntax for consistency.